src/DynamoCoreWpf/Controls/NotificationsControl.xaml.cs

```diff
@@ -40,9 +40,14 @@ void NotificationsControl_Loaded(object sender, RoutedEventArgs e)
         /// <param name="e"></param>
         void window_PreviewMouseDown(object sender, MouseButtonEventArgs e)
         {
-            var hsvm = (HomeWorkspaceViewModel)((DynamoViewModel)DataContext).HomeSpaceViewModel;
+            var hsvm = (HomeWorkspaceViewModel)((DynamoViewModel)DataContext).HomeSpaceViewModel;            
             // Commented this after MAGN - 8423
-            // hsvm.ClearWarning();
+            //hsvm.ClearWarning();
+            if (hsvm.IsHomeSpace && !hsvm.IsCurrentSpace)
+            {
+                hsvm.DynamoViewModel.FileTrustViewModel.ShowWarningPopup = false;
+                hsvm.ClearWarning();
+            }
```

Comment:
hmm why do we need to touch this piece of code?

Reply:
it forces to hide the untrusted message and bubble when we start to work in a just created custom node workspace, not only when we switch the workspaces.